src/App.js

```diff
@@ -1,9 +1,9 @@
 import React from 'react';
 import './css/main.css';
-import { LessonInfo } from './components';
+import CatalogPage from './components/catalog/CatalogPage';
```

Comment:
This will get removed anyway when not using App to immediately load the specific component, but remember to make use of our `index.js` file.
```suggestion
import { CatalogPage } from './components';
```